altanalyze3/utilities/parser.py

```diff
@@ -147,6 +149,35 @@ def get_parser(self):
             action="store_true"
         )
         self.add_common_arguments(intron_parser)
+
+        # Protein Domain Annotation parser
+        protein_coordinates_parser = subparsers.add_parser(
+            "proteincoordinates",
+            parents=[parent_parser],
+            help="Get Protein to Domain annotations"
+        )
+        protein_coordinates_parser.set_defaults(func=get_protein_coordinates)
+        protein_coordinates_parser.add_argument(
+            "--name",
+            help="name of species eg. apolyacanthus_gene_ensembl",
+            type=str,
+            required=True,
```

Comment:
You can use `choices` here

Reply:
Isn't choices meant for giving users some limited options?

If we do it this way, the user might get confused and think they can select species only from this given menu. Help should be enough in this case.